packages/@sanity/desk-tool/src/panes/documentsListPane/DocumentsListPane.js

```diff
@@ -192,14 +192,17 @@ export default class DocumentsListPane extends React.PureComponent {
   }
 
   renderItem = (item) => (
-    <PaneItem
-      id={getPublishedId(item._id)}
-      layout={this.state.layout || this.props.defaultLayout || 'default'}
-      value={item}
-      icon={this.props.displayOptions.showIcons === false ? false : undefined}
-      schemaType={schema.get(item._type)}
-      isSelected={this.itemIsSelected(item)}
-    />
+    <Box paddingX={2} paddingBottom={0}>
+      <PaneItem
+        id={getPublishedId(item._id)}
+        layout={this.state.layout || this.props.defaultLayout || 'default'}
+        value={item}
+        icon={this.props.displayOptions.showIcons === false ? false : undefined}
+        schemaType={schema.get(item._type)}
+        isSelected={this.itemIsSelected(item)}
+        isActive
```

Comment:
what's the difference between `isSelected` and `isActive`? in this context, why is `isActive` hard-coded true?

Reply:
Active pane items have different background and foreground 👍